src/courtroom.cpp

```diff
@@ -2798,6 +2799,8 @@ void Courtroom::start_chat_ticking()
 
   QString f_gender = ao_app->get_gender(m_chatmessage[CHAR_NAME]);
 
+  gen_char_rgb_list(m_chatmessage[CHAR_NAME]);
```

Comment:
Wait, this generates the char_rgb_list every time there's a new message??? That sounds expensive...

Reply:
I've not noticed a significant performance hit, but if performance is a concern it'd be possible to only generate the list if the character's misc changes

Reply:
Let's do that

Reply:
Done.